package.json

```diff
@@ -13,9 +13,10 @@
     "test-jest-cov": "yarn run test-jest --coverage",
     "coverage": "yarn run test-jest-cov",
     "build": "kolibri-tools build prod --file ./build_tools/build_plugins.txt && yarn run hashi-build",
-    "makemessages": "kolibri-tools build i18n --file ./build_tools/build_plugins.txt",
+    "makemessages": "NODE_ENV=i18n-extract kolibri-tools build i18n --file ./build_tools/build_plugins.txt",
+    "profilemessages": "NODE_ENV=i18n-profile kolibri-tools build i18n --file ./build_tools/build_plugins.txt",
     "combineprofiles": "node ./packages/kolibri-tools/lib/combineStringProfiles.js ./kolibri/locale/en/LC_MESSAGES/profiles/",
-    "transfercontext": "node ./packages/kolibri-tools/lib/i18nSyncContext.js run && yarn lint-frontend:format",
+    "transfercontext": "APP_NAME=kolibri node ./packages/kolibri-tools/lib/i18nSyncContext.js run && yarn lint-frontend:format",
```

Comment:
Rather than controlling `kolibri-tools build` behavior with env vars and setting them like this, maybe we should instead have different commands for each action.

If necessary we can still still provide need a generic, env-var-driven `build` wrapper command

Reply:
Yeah, I'm not a big fan of the env vars here, would rather add more command line flags, or additional commands.

I think in general, we could pull the `i18n` command out into its own top level command in `kolibri-tools`, it was only in `build` historically because of the use of the webpack pipeline to do string extraction. I've not looked over the PR in detail yet, but my assumption was we were dropping that for pure AST parsing.